identity/common/src/main/java/io/camunda/identity/user/Group.java

```diff
@@ -5,7 +5,7 @@
  * Licensed under the Camunda License 1.0. You may not use this file
  * except in compliance with the Camunda License 1.0.
  */
-package io.camunda.identity.usermanagement;
+package io.camunda.identity.user;
```

Comment:
The group package would contain at least the Group DTO and a group service class. Depending on how we move forward with the database usage/implementation it could also contain a group repository. Also there will be a group controller